pulpcore/pulpcore/app/viewsets/repository.py

```diff
@@ -137,11 +137,17 @@ def content(self, request, repository_pk, number):
         return self._paginated_response(self.get_object().content(), request)
 
     @decorators.detail_route()
-    def added(self, request, repository_pk, number):
+    def added_content(self, request, repository_pk, number):
+        """
+        Display content added since the previous Repository Version.
```

Comment:
This is displayed in the api schema. See https://pulp.plan.io/issues/3283 for more info.